solidity/contracts/deposit/DepositLiquidation.sol

```diff
@@ -180,7 +180,7 @@ library DepositLiquidation {
 
         // Distribute funds to auction buyer
         uint256 _valueToDistribute = _d.auctionValue();
-        msg.sender.transfer(_valueToDistribute);
+        msg.sender.call.value(_valueToDistribute)("");
```

Comment:
As above re: checks-effects-interactions.

Reply:
`_d.setLiquidated();` is at the top of the function, preventing reentrancy.

Not really much more to move up aside from the initiator check.